WordPress/UITestsFoundation/Screens/CommentsScreen.swift

```diff
@@ -0,0 +1,63 @@
+import ScreenObject
+import XCTest
+
+public class CommentsScreen: ScreenObject {
+
+    private let replyTextViewGetter: (XCUIApplication) -> XCUIElement = {
+        $0.textViews["ReplyText"]
+    }
+
+    private let backButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.navigationBars.buttons["Reader"]
+    }
+
+    private let replyButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.buttons["Reply"]
+    }
+
+    var replyTextView: XCUIElement { replyTextViewGetter(app) }
+    var backButton: XCUIElement { backButtonGetter(app) }
+    var replyButton: XCUIElement { replyButtonGetter(app) }
+
+    init(app: XCUIApplication = XCUIApplication()) throws {
+        try super.init(
+            expectedElementGetters: [
+                // swiftlint:skip:next opening_brace
+                { $0.navigationBars["Comments"] },
+                replyTextViewGetter
+            ],
+            app: app,
+            waitTimeout: 7
+        )
+    }
+
+    public static func isLoaded() -> Bool {
+        (try? ReaderScreen().isLoaded) ?? false
+    }
+
+    public func navigateBack() throws -> ReaderScreen {
+        backButton.tap()
+        return try ReaderScreen()
+    }
+
+    @discardableResult
+    public func replyPost(_ comment: String) -> CommentsScreen {
+        app.otherElements.containing(.staticText, identifier: "Reply to post").lastMatch?.tap()
+        replyTextView.typeText(comment)
+        replyButton.tap()
+        return self
+    }
+
+    public func verifyCommentsListEmpty() -> CommentsScreen {
+        XCTAssertTrue(app.tables.firstMatch.label == "Empty list")
+        XCTAssertTrue(app.staticTexts["Be the first to leave a comment."].isHittable)
```

Comment:
same as previous comment, we could benefit from using accessibility identifiers here too